src/scriptworker/constants.py

```diff
@@ -365,12 +376,18 @@
                             "project:mobile:fenix:releng:signing:cert:nightly-signing": "fenix-repo",
                             "project:mobile:fenix:releng:signing:cert:beta-signing": "fenix-repo",
                             "project:mobile:fenix:releng:signing:cert:production-signing": "fenix-repo",
+                            "project:mobile:fenix:releng:signing:cert:fennec-nightly-signing": "fenix-repo",
+                            # Fennec Beta and Fennec Release share the same key
+                            "project:mobile:fenix:releng:signing:cert:fennec-production-signing": "fenix-repo",
+                            "project:mobile:fennec-profile-manager:releng:signing:cert:fennec-nightly-signing": "fenix-repo",
```

Comment:
`s/fenix-repo/fennec-profile-manager`for this line and the next in their values? I'm assuming we want to be the value we defined at https://github.com/mozilla-releng/scriptworker/pull/433/files#diff-4635e904df4496f16271f8684ac489f5R475

Reply:
👍 Fixed